pkg/cli/build.go

```diff
@@ -263,6 +272,7 @@ func buildCmd() *cobra.Command {
 	cmd.Flags().StringVar(&configFileGitCommit, "git-commit", "", "commit hash of the git repository containing the build config file (defaults to detecting HEAD)")
 	cmd.Flags().StringVar(&configFileGitRepoURL, "git-repo-url", "", "URL of the git repository containing the build config file (defaults to detecting from configured git remotes)")
 	cmd.Flags().StringVar(&configFileLicense, "license", "NOASSERTION", "license to use for the build config file itself")
+	cmd.Flags().StringSliceVar(&volumeMounts, "volumes", []string{}, "bind mount a volume(s) into the container (e.g., /host:/container)")
```

Comment:
I'd this `volume` and do `StringSliceVarP` so we can use `-v` (to mirror docker).